### DISCUSSION TITLE
What is the average human body temperature when taken by a thermometer under the tongue?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#185] [Galaxy Trivia] [answered]
**Hint:** 🤒

**Comment by thenithinbalaji** [comment]
The average body temperature is 98.6 F (37 C). But normal body temperature can range between 97 F (36.1 C) and 99 F (37.2 C) or more.

**Comment by thuongtruong1009** [comment, marked as answer]
The average human body temperature when taken by a thermometer under the tongue (oral temperature) is around 98.6 degrees Fahrenheit (37 degrees Celsius). However, it's important to note that this is just an average, and normal body temperature can vary slightly from person to person and can also fluctuate throughout the day.
